### PR TITLE
Update README, cleanup

### PR DESCRIPTION
**Pull request by lslezak**
- I'm not merging to `master` but to a separate branch so we can do the review in smaller parts and merge to `master` when ready to submit the first version.
- Updated README
- Small cleanup
- Look for "SLE*" migrations, in theory there might be also a migration to "SLED"
- More will follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,113 @@
...
+#### Rollback
+
+When the migration fails or is aborted then YaST restores the original
+repositories in the system and moves the registration from SLE back to openSUSE
+Leap. However, the system still remains in a registered state.
+
+If you want to fully unregister the system then run this command as `root`:
+
+```shell
+SUSEConnect --de-register
```
Comment (dgdavid) [inline README.md:71]: Question: why isn't it done as part of the rollback process?

**Review by dgdavid — APPROVED** [review]
LGTM, just a question out of curiosity.

File: README.md (hunk 1 above)
Comment (lslezak) [reply README.md:71]: Unfortunately the [YaST SUSEConnect API](https://github.com/SUSE/connect-ng/blob/main/yast/lib/suse/connect/yast.rb) does not export functionality for that... :worried: 
Later I'll request that.

Comment (dgdavid) [reply README.md:71]: Ah, ok :sob:  Thanks!

**Pull request merged**